packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/sagas.ts

```diff
@@ -418,7 +418,12 @@ export default ({ api, coreSagas, networks }) => {
             break
```

Comment:
do you need to add that logic here as well?

Reply:
in general we can if we want to prevent additional calls if saga trigger 10 times, I'm not sure is it possible to happen that we trigger 10 times and BE is still not really processed this we mark it and then when user enter other page we do call to BE it user is verified now. I think if we add same logic we might lost this edge case.